resources/js/processes/modeler/components/inspector/TaskAssignment.vue

```diff
@@ -36,19 +37,20 @@
 
         <form-checkbox
             :label="$t('Allow Reassignment')"
+            :helper="$t('Allows the Task assignee to reassign this Task')"
             :checked="allowReassignmentGetter"
             @change="allowReassignmentSetter">
         </form-checkbox>
 
         <div class="form-group">
 
             <div class="form-group special-assignment-header">
-                <label>{{ $t("Assign by Expression") }}</label>
+                <label>{{ $t("Assign by Expression: Use a rule to assign this Task conditionally") }}</label>
```

Comment:
Wondering if this is why it says:
"Missing Translation:  Use a rule to assign this Task conditionally"

I don't see "Use a rule to assign this Task conditionally" used anywhere on its own here.